CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## UNRELEASED
 
+IMPROVEMENTS:
+
+* Connect: Add resource request and limit flags for the injected init container and lifecycle sidecar.
```

Comment:
```suggestion
* Connect: Add resource request and limit flags for the injected init and lifecycle sidecar containers [[GH-298](https://github.com/hashicorp/consul-k8s/pull/298)].
```

Reply:
Should we also mention what their defaults are and why?

Reply:
Updated. No, I don't think listing the new flags is necessary here. The flags are clearly laid out in the PR and descriptions available through the command's help text.

The pull request didn't exist so I couldn't link it when I wrote this up. It's pretty circular to expect it as part of the PR submission.